bsp_diff/caas/device/intel/civ/host/vm-manager/0021-Revert-Add-config-ini-as-part-of-package.patch

```diff
@@ -0,0 +1,84 @@
+From a3975e4d8eccd1792b472d8c3823e605872cf220 Mon Sep 17 00:00:00 2001
+From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
+Date: Mon, 10 Apr 2023 20:04:19 +0530
+Subject: [PATCH 3/7] Revert "Add config ini as part of package."
```

Comment:
This seems to be reverting the vertical patch 0005-Add-config-ini-as-part-of-package.patch.
And there is another patch 0023-Add-config-ini-as-part-of-package.patch to add it back.
Please check.